proxy/logging/LogBuffer.cc

```diff
@@ -112,14 +112,6 @@ LogBufferHeader::log_filename()
   return addr;
 }
 
-/*-------------------------------------------------------------------------
-  LogBuffer::LogBuffer
```

Comment:
I talked with @danobi about this and my view is a local comment should be minimal and the real meat in the Architecture section of the documentation. This commentary has obviously not been maintained and I don't see it being any different in the future. At least in the normal documentation there's some hope of it being updated.